### ISSUE TITLE
Option to have searchword suggestions in explore mode

### ISSUE THREAD

**Issue by ottopiramuthu** [Jelmerro/Vieb#167]
**Checklist**
- [x] I searched the `:help` documentation using `/` for helpful information
- [x] I have read the [frequently asked questions](https://github.com/Jelmerro/Vieb/blob/master/FAQ.md)
- [x] I did not find any similar issues in the [issue tracker](https://github.com/Jelmerro/Vieb/issues)

**Addition or change**
Once you make many searchwords, the ones you use less often become hard to remember. It would be nice to have an option to show searchwords suggestions in explore mode.
Normally, explore mode suggests past searches, urls in history, and files:
![image](https://user-images.githubusercontent.com/49997309/115973949-dbce4200-a526-11eb-9f9a-7b6fab45173d.png)
Let's say I have this line in my viebrc: `set searchwords+=dbse~https://dba.stackexchange.com/search?q=%s`.
Then in addition to the suggested urls, Vieb would have another suggestion like:
`dbse         https://dba.stackexchange.com/search?q=%s` which you could select to change the navbar's text to `dbse `.
 

**Alternatives considered**
Remembering the searchwords: gets pretty hard when you have a lot of them most of which you rarely use.

**Comment by Jelmerro** [comment]
In [this comment](https://github.com/Jelmerro/Vieb/issues/100#issuecomment-812579968), among other things, I suggested the `suggestorder` setting. I think it would be perfect to implement such a setting to allow more customization for the suggestions in explore mode. Along with "history", "files" and "bookmarks", we could just add a supported entry to the list for "searchwords", which can then also be added to the list the order you want to them to. In terms of sorting the searchwords, we could do it based on "alpha", "visits" (based on history) and a new sort order, which is the order that they appear in the searchwords setting list. Though I'm not sure how to call that yet.

As I don't really use bookmarks myself, I'll probably pick this up first and rework the suggestions to make use of a new `suggestorder` setting as proposed, which will also reduce the amount of work for potential contributors to the bookmarks ticket. Although keep in mind that creating this `suggestorder` setting and removing other outdated settings is a breaking change, so I'll probably wait until I have a bit more stuff I want to fundamentally change before working on this.

**Comment by ottopiramuthu** [comment]
`suggestorder` with `searchwords` looks like exactly what I want. Looking forward to this change once Vieb is ready.

**Comment by Jelmerro** [comment]
Implemented in [V5](https://github.com/Jelmerro/Vieb/releases/tag/5.0.0) and enabled by default, see `:h suggestorder` for details.
